<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>541</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Фильтр</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="filter1Label">
       <property name="text">
        <string>Фильтр 1:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="filter1LineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="filter2Label">
       <property name="text">
        <string>Фильтр 2:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="filter2LineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="filter3Label">
       <property name="text">
        <string>Фильтр 3:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="filter3LineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="filter4Label">
       <property name="text">
        <string>Фильтр 4:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="filter4LineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="filter5Label">
       <property name="text">
        <string>Фильтр 5:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="filter5LineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="selectFolderButton">
       <property name="text">
        <string>Выбрать папку:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="selectFolderLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="filterButton">
     <property name="text">
      <string>Отфильтровать</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="filterTextEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
